### PR TITLE
feat: add virgo to packages

### PR DESCRIPTION
**Pull request by Flrande**

### CODE REVIEW

**Review by Himself65 — CHANGES_REQUESTED** [review]
https://github.com/toeverything/blocksuite/blob/master/tsconfig.json
https://github.com/toeverything/blocksuite/blob/master/.eslintrc.cjs
https://github.com/toeverything/blocksuite/blob/master/scripts/publish-to-github.sh

need update

File: packages/virgo/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export * from './Editor.js';
```
Comment (himself65) [inline packages/virgo/src/index.ts:2]:
```suggestion
/// <reference types="@blocksuite/global" />
export * from './Editor.js';
```

**Review by Himself65 — CHANGES_REQUESTED** [review]

File: packages/virgo/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@blocksuite/virgo",
+  "version": "0.1.0",
```
Comment (himself65) [inline packages/virgo/package.json]: 0.4.0-alpha.0

**Review by Himself65 — CHANGES_REQUESTED** [review]

File: packages/virgo/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "devDependencies": {
+    "yjs": "^13.5.44",
+    "lit": "^2.6.1"
+  },
+  "peerDependencies": {
+    "yjs": "^13",
+    "lit": "^2"
+  },
+  "exports": {
+    "./src/*": "./src/*.ts",
```
Comment (himself65) [inline packages/virgo/package.json]:
```suggestion
    "./*": "./src/*.ts",
```

File: packages/virgo/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    ".": "./src/index.ts"
+  },
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "exports": {
+      "./src/*": "./dist/*.js",
```
Comment (himself65) [inline packages/virgo/package.json]:
```suggestion
      "./*": "./dist/*.js",
```

File: packages/virgo/playwright.config.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      (process.env.BROWSER as PlaywrightWorkerOptions['browserName']) ??
+      'chromium',
+    viewport: { width: 900, height: 600 },
+    actionTimeout: 1000,
+  },
+};
+
+if (process.env.CI) {
+  config.webServer = {
+    command: 'pnpm dev',
```
Comment (himself65) [inline packages/virgo/playwright.config.ts]: no dev here

**Review by Himself65 — CHANGES_REQUESTED** [review]

**Review by Himself65 — CHANGES_REQUESTED** [review]
https://github.com/toeverything/blocksuite/blob/master/.commitlintrc.json

**Review by Himself65 — APPROVED** [review]

**Pull request merged**